arraycontext/impl/pytato/fake_numpy.py

```diff
@@ -41,7 +40,7 @@ class PytatoFakeNumpyLinalgNamespace(BaseFakeNumpyLinalgNamespace):
     pass
 
 
-class PytatoFakeNumpyNamespace(BaseFakeNumpyNamespace):
+class PytatoFakeNumpyNamespace(LoopyBasedFakeNumpyNamespace):
```

Comment:
Should `PytatoFakeNumpyNamespace` inherit from `LoopyBasedFakeNumpyNamespace`? It seems to implement its own `__getattr__`.

Reply:
As a fallback, call_loopy for these ufuncs is fine IMO.

Reply:
Yeah, that makes sense to me. Just thought I'd raise the issue. Thanks!